service/service.py

```diff
@@ -107,7 +107,7 @@ def delete_a_supplier(supplierID):
     """ Route to delete a supplier """
     supplier = Supplier.find(supplierID)
     if supplier:
-        supplier.delete()
+        supplier.delete(supplierID)
```

Comment:
**supplier = Supplier.find(supplierID)** 
And **supplier.delete()** here called a built-in MongoEngine function that does not require any parameter.

I don't know if we should change to call the local **delete()** function that I implemented in  [this commit](https://github.com/nyu-devops-fall19-suppliers/suppliers/commit/4ee02286023fa006263b47b571b3909fe826bb28#diff-96373c14f7e2c24a0689a6c09cfb9fe2)

Anyway I prefer to preserve the global one though since my implementation seems redundant here.

Reply:
Because an error said delete() missing an argument, I just add it here. But does not work... I will remove the argument and dive deep on this.

Reply:
Then try to completely remove the redundant local function I defined?